Autopilot/Path Manager/PathManager.h

```diff
@@ -58,6 +58,8 @@ void copyGPSData(void);
 char gpsErrorCheck(double lat, double lon);
 void checkAMData(void);
 float getWaypointChecksum(void);
+void GPSLock();
+char checkForReset();
```

Comment:
No need for either of these functions to be declared inside the header, since they are only internally used within the Path Manager. These functions should be declared static so that they only exist within the scope of the PathManager.cpp file, and removed from the header.